### DISCUSSION TITLE
Orientation in device manager

### DISCUSSION THREAD

**Discussion by shubhampalgaian** [GrapesJS/grapesjs#5493] [Q&A] [answered]
Is there any way we can add orientations in device manager and there related functions that an be used to handle the functionalities ?

**Comment by mosh-tudor** [comment]
> Is there any way we can add orientations in device manager

Check
https://grapesjs.com/docs/api/device_manager.html

**Reply by shubhampalgaian** [reply]
there is nothing related to orientations in that page to my knowledge. ? can you confirm with the some screenshot or exact where i can find things about orientations

**Reply by mosh-tudor** [reply, marked as answer]
You have to manually add "orientations" as new devices.

**Reply by shubhampalgaian** [reply]
yes i have done that way only.  i was looking if grapesjs provide something related to that. thanks for the suggestion. appreciate.
